### PR TITLE
SPCTR-784 : Blockquote Fixes (R3)

### PR DESCRIPTION
**Pull request by aapatre**
### Description

1. In the Blockquote, the Stacked Twitter Gap control does not work as expected. Also we need to replace the current label string with something relevant & easily consumable
2. If the author image has larger height, it can appear squashed/stretched
3. We must categories all controls properly. The controls which affect the HTML should only get added to the General tab. Other controls which are responsible for the styling of the block should get added to the Style tab.
4. In the Blockquote block, the label Author Image Rounded Corners (in settings) seems too long and creates UI conflicts. Ideally, it should get replaced with the string Image Border Radius. 

### JIRA
https://brainstormforce.atlassian.net/browse/SPCTR-784

### Types of changes

Bug fix (non-breaking change which fixes an issue)

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Comment by aapatre** [comment]
JIRA Issue: https://brainstormforce.atlassian.net/browse/SPCTR-784

**Review by bsf-zanev — APPROVED** [review]
All code removals and additions are clean and required.

File: src/blocks/blockquote/settings.js [hunk 1]
```diff
@@ -1706,12 +1684,13 @@ const Settings = ( props ) => {
...
 						{ quoteStyling }
 						{ authorStyling }
-						{ spacingSettings }
 					</InspectorTab>
 					<InspectorTab
 						{ ...UAGTabs.advance }
 						parentProps={ props }
-					></InspectorTab>
+					>
+						{ spacingSettings }
```
Comment (supriyajjadhav) [inline src/blocks/blockquote/settings.js:1692]: You have added spacing panel in Advanced Tab. Please check

Comment (aapatre) [reply src/blocks/blockquote/settings.js:1692]: Yes, as per the new style guide we need to move spacing panel to the Advanced Tab.

**Review by bsf-zanev — APPROVED** [review]
New Changes Viewed and Approved.

File: src/blocks/blockquote/settings.js (hunk 1 above)
Comment (RajkiranBagal) [reply src/blocks/blockquote/settings.js:1692]: @aapatre We have not started working on the style guide yet. Keep things as it is for now. We will have a separate task for the Style Guide changes, So in that way we will have consistent settings in all blocks.

**Review by RajkiranBagal — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**